lms/djangoapps/instructor/views/api.py

```diff
@@ -3391,13 +3414,15 @@ def update_cohort_assignment(request, course_id):
         cohort_for_change = [cohort_id]
 
     if is_assignment:
-        cohort_to_incert = [CohortAssigment(cohort_id=cohort, user=user) for cohort in cohort_for_change]
-        CohortAssigment.objects.bulk_create(cohort_to_incert)
-    else:
-        CohortAssigment.objects.filter(cohort__id__in=cohort_for_change, user=user).delete()
+        created = False
+        for cohort in cohort_for_change:
+            if not cohort:
```

Comment:
Let's avoid it by not adding None objects into `cohort_for_change` list.

Reply:
Add a fix. Thanks to that you mention it to me.